src/SheetsRegistry.js

```diff
@@ -15,7 +15,21 @@ export default class SheetsRegistry {
    * @api public
    */
   add(sheet) {
-    this.registry.push(sheet)
+    const {registry} = this
+
+    if (registry.length && sheet.options.index) {
```

Comment:
@kof Yep, I will fix this -- I had initially set this function up using `1+` input before our prev discusson regarding valid range.